utils/vlog/vlog_to_pbtype.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+import yosys_exec
+import lxml.etree as ET
+import argparse, re
+import os, tempfile
+from yosys_json import YosysJson
+
+"""
+Convert a Verilog simulation model to a VPR `pb_type.xml`
```

Comment:
None of these attributes are in your tests?

Reply:
The tests at the moment are just for the model.xml generation, still working on examples for the pb_type stuff